### PR TITLE
updated swagger doc

### PR DESCRIPTION
**Pull request by rall**
Updated api documentation. The provider and insurance entities are gone, munged into appointment and user respectively.

### CODE REVIEW

File: public/swagger.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
         "title": "Vitals Online Appointments"
     },
     "schemes": [
-        "http",
```
Comment (waysidekoi) [inline public/swagger.json]: Why drop http?

Comment (rall) [inline public/swagger.json]: should we keep?

Comment (rall) [inline public/swagger.json]: that was rather a casual deletion :-)

**Comment by waysidekoi** [comment]
👍

**Pull request merged**